src/components/Form/SetAclPermsDocContainerForm.jsx

```diff
@@ -56,37 +50,21 @@ const SetAclPermsDocContainerForm = () => {
     try {
       await setDocContainerAclPermission(session, permissions, podUrl, webIdToSetPermsTo);
 
-      runNotification(
-        `${permissions.read ? 'Give' : 'Revoke'} permission to ${
-          permissionState.podUrlToSetPermissionsTo
-        } for Documents Container.`,
-        5,
-        state,
-        dispatch
-      );
       addNotification(
         'success',
         `${permissions.read ? 'Gave' : 'Revoked'} permission to ${
           permissionState.podUrlToSetPermissionsTo
         } for Documents Container.`
       );
     } catch (error) {
-      runNotification('Failed to set permissions. Reason: File not found.', 5, state, dispatch);
       addNotification('error', 'Failed to set permissions. Reason: File not found.');
```

Comment:
Not sure if this "Reason" is correct.

Reply:
Yeah, the reason for this error is not correct. This is the error statement for failing to set permissions for the Documents container. For now, I think we could use `error.message`.